### ISSUE TITLE
Helm Repo not working

### ISSUE THREAD

**Issue by derjohn** [k8up-io/k8up#741]
### Description

I think the repo is missing a index.yaml .....

### Expected Behavior

helm repo working

### Version of K8up

main branch

### Version of Kubernetes

9.99

### Distribution of Kubernetes

Windowsx 3.11 for Workgroups

**Comment by derjohn** [comment]
helm from shell works, with the terraform helm provider it doesnt.

**Comment by sashokbg** [comment]
Hello @derjohn you would need to provide more information on exactly does not work. Do you have any error messages or logs that you can share ?
Also have you tried looking at the state of your helm chart ?
Did you check pod logs etc ?

**Comment by Kidswiss** [comment]
Hi @derjohn 

I can only agree to what @sashokbg already said. It's hard to determine what's not working without any further context. Can you at least add some errors or logs?

The `index.yaml` file is available and reachable as far as I'm able to determine:
```
curl https://k8up-io.github.io/k8up/index.yaml
apiVersion: v1
entries:
  k8up:
  - apiVersion: v2
    created: "2022-09-15T09:18:26.400474787Z"
    description: Kubernetes and OpenShift Backup Operator based on restic
    digest: 3143e7518441ddd079d819785d06d11968ce79d63f208b2412e3368129b955fc
    home: https://k8up.io/
    keywords:
    - backup
    - operator
    - restic
    maintainers:
    - email: info@appuio.ch
      name: K8up Authors
....
```

**Comment by derjohn** [comment]
Uhm, I closed the issue yesterday= I didn't? oops ;)

I can say that "helm repo add ...." on the shell works with helm3, but if I do so with terraform:

```
resource "helm_release" "k8up" {
  name            = "k8up"
  repository     = "https://k8up-io.github.io/k8up"
  chart             = "k8up"
  namespace  = ...
}
```

results in an error. Usually helm repos have an index.yaml ( https://helm.sh/docs/helm/helm_repo_index/ ) which describes the charts inside the repo. It looks like the helm provider in terraform does not support helm repos without index.yaml.

As a workaround I checked out the k8uo githubrepo and went and set chart = .. to a local file path.
